### PR TITLE
feat: updates to the site about page and faq

### PR DESCRIPTION
**Pull request by cpp6d**
Updates the FAQ and About page and minor changes to the nav

### CODE REVIEW

File: _includes/navigation.html [hunk 1]
```diff
@@ -15,12 +15,12 @@
...
-            <li class="nav-item{% if include.active == "download" %} active{% endif %}">
-                <a class="nav-link py-2 py-lg-4 px-2 px-xl-3" href="{{ site_prefix }}/download">{{ strings.navigation.download }}</a>
+            <li class="nav-item{% if include.active == "github" %} active{% endif %}">
+                <a class="nav-link py-2 py-lg-4 px-2 px-xl-3" href="https://github.com/{{ site.data.contact.github_username }}" target="_blank" rel="noreferrer noopener">GitHub</a>
                 <div class="highlight d-none d-lg-inline-block"></div>
             </li>
-            <li class="nav-item{% if include.active == "blog" %} active{% endif %}">
-                <a class="nav-link py-2 py-lg-4 px-2 px-xl-3" href="{{ site_prefix }}/blog">{{ strings.navigation.blog }}</a>
+            <li class="nav-item{% if include.active == "download" %} active{% endif %}">
+                <a class="nav-link py-2 py-lg-4 px-2 px-xl-3" href="{{ site_prefix }}/">{{ strings.navigation.download }}</a>
```
Comment (hoffmang9) [inline _includes/navigation.html:23]: The non GitHub/Download page stuff looks fine, but why are we adding GitHub to the Nav bar and removing the well craft and automated Download pate?

Comment (cpp6d) [reply _includes/navigation.html:23]: Per the request of @roybotbot He wanted to add the GitHub page on the far left and remove the Blog tab. Which moved the download page to the right 1 slot

**Comment by cpp6d** [comment]
> @cpp6d you can delete this PR. My mistake. This change was meant for Chialisp.com

We still need this pr for the other changes, but I removed those changes you requested.

**Review by sargonas — DISMISSED** [review]
remaining changes are exactly what I need, thanks!

**Pull request closed without merging**